server/schedulers/evict_leader.go

```diff
@@ -36,8 +36,11 @@ const (
 	// EvictLeaderName is evict leader scheduler name.
 	EvictLeaderName = "evict-leader-scheduler"
 	// EvictLeaderType is evict leader scheduler type.
-	EvictLeaderType     = "evict-leader"
-	lastStoreDeleteInfo = "The last store has been deleted"
+	EvictLeaderType = "evict-leader"
+	// EvictLeaderBatchSize is the number of operators to to transfer
+	// leaders by one scheduling
+	EvictLeaderBatchSize = 3
```

Comment:
How about making it configurable?

Reply:
Leader transferring is a very light operation with a tiny impact on the performance, there is little necessity for a user to customize it.